tbx/project_styleguide/templates/patterns/base.html

```diff
@@ -1,6 +1,6 @@
 {% load static wagtailuserbar wagtailcore_tags wagtailimages_tags navigation_tags util_tags wagtailaccessibility_tags %}
 <!doctype html>
-<html class="{% block body_class %}template__{{ page.get_verbose_name|slugify }}{% endblock %}" lang="en-GB">
+<html class="{% block body_class %}template-{{ page.get_verbose_name|slugify }}{% endblock %}" lang="en-GB">
```

Comment:
I'm not sure why this was in BEM format - could just be leftover from the current site but updated to how we do it in WK. (I can't find anything in the codebase for `template__`)